### DISCUSSION TITLE
Projects (beta): Link Field

### DISCUSSION THREAD

**Discussion by mcwhittemore** [community/community#6574] [Projects and Issues]
It would be nice to be able to have a field type that takes and validates URLs. Think could be used to link our to figma designs or used by meta project boards to link out a view of the tasks covered by a line item.

**Comment by galargh** [comment]
It would be great if the link field acted as a link in the board view too. Right now, if you use a text field to store links and click on them in the board view, they just set the board filter rather than redirecting to the URL.

**Reply by galargh** [reply]
I've just found out if you use HTML inside the text field, it is rendered in the views. So to get a clickable link in the board view, one can set the test to something like `<a href="URL">LINK</a>`.
